src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/IndexPageTests.kt

```diff
@@ -112,4 +111,38 @@ class IndexPageTests : SeleniumTest()
         val component = driver.findElements(By.cssSelector("#setPinnedReportsVueApp"))
         assertThat(component.count()).isEqualTo(0)
     }
+
+    @Test
+    fun `can link to run report page with permission`()
+    {
+        setUpDb()
+        startApp("auth.provider=montagu")
+
+        addUserWithPermissions(listOf(
+                ReifiedPermission("reports.run", Scope.Global())))
+
+        loginWithMontagu()
+        driver.get(RequestHelper.webBaseUrl)
+
+        val component = driver.findElements(By.cssSelector(".ml-5 > a:nth-child(1)"))
+        assertThat(component.count()).isEqualTo(1)
+        component[0].click();
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("#git-branch")))
```

Comment:
This obviously works fine, but it we switched to a non-git model it would fail. Could just check whether the browser navigates to the run-report page.